undefined: drivermanager.TestDriverManager during `make release`

Following the release instructions causes this error:

```
$ make release
==> Cleaning build artifacts...
==> Building pkg/darwin_amd64/nomad with tags ui release...
# github.com/hashicorp/nomad/client/allocrunner
client/allocrunner/testing.go:66:22: undefined: drivermanager.TestDriverManager
make: *** [pkg/darwin_amd64/nomad] Error 2
```

From a clean install:

```
~/.go/src/github.com/hashicorp $ git clone https://github.com/hashicorp/nomad
Cloning into 'nomad'...
remote: Enumerating objects: 92, done.
remote: Counting objects: 100% (92/92), done.
remote: Compressing objects: 100% (55/55), done.
remote: Total 103771 (delta 51), reused 55 (delta 37), pack-reused 103679
Receiving objects: 100% (103771/103771), 117.34 MiB | 11.96 MiB/s, done.
Resolving deltas: 100% (73301/73301), done.
~/.go/src/github.com/hashicorp $ cd nomad/
~/.go/src/github.com/hashicorp/nomad $ make bootstrap
==> Updating build dependencies...
go get -u github.com/kardianos/govendor
go get -u github.com/ugorji/go/codec/codecgen
go get -u github.com/hashicorp/go-bindata/go-bindata
go get -u github.com/elazarl/go-bindata-assetfs/go-bindata-assetfs
go get -u github.com/a8m/tree/cmd/tree
go get -u github.com/magiconair/vendorfmt/cmd/vendorfmt
go get -u github.com/golang/protobuf/protoc-gen-go
==> Updating linter dependencies...
go get -u github.com/alecthomas/gometalinter
gometalinter --install
Installing:
  deadcode
  dupl
  errcheck
  gochecknoglobals
  gochecknoinits
  goconst
  gocyclo
  goimports
  golint
  gosec
  gosimple
  gotype
  gotypex
  ineffassign
  interfacer
  lll
  maligned
  megacheck
  misspell
  nakedret
  safesql
  staticcheck
  structcheck
  unconvert
  unparam
  unused
  varcheck
~/.go/src/github.com/hashicorp/nomad $ make release
==> Cleaning build artifacts...
==> Building pkg/darwin_amd64/nomad with tags ui release...
# github.com/hashicorp/nomad/client/allocrunner
client/allocrunner/testing.go:66:22: undefined: drivermanager.TestDriverManager
make: *** [pkg/darwin_amd64/nomad] Error 2
```
`make dev` works though:
```
$ make dev
--> Formatting vendor/vendor.json
test -x /Users/rjones/.go/bin/vendorfmt || go get -u github.com/magiconair/vendorfmt/cmd/vendorfmt
vendorfmt
Formatted vendor/vendor.json
--> Making [GH-xxxx] references clickable...
==> Removing old development build...
==> Building pkg/darwin_amd64/nomad with tags ...
~/.go/src/github.com/hashicorp/nomad $ ls pkg/darwin_amd64/
nomad
```

Removing the testing file gives:

```
$ make release
==> Cleaning build artifacts...
==> Building pkg/darwin_amd64/nomad with tags ui release...
# github.com/hashicorp/nomad/command/agent
command/agent/bindata_assetfs.go:29:2: imported and not used: "io/ioutil"
command/agent/bindata_assetfs.go:31:2: imported and not used: "path/filepath"
command/agent/bindata_assetfs.go:32:2: imported and not used: "strings"
command/agent/bindata_assetfs.go:35:2: imported and not used: "github.com/hashicorp/nomad/vendor/github.com/elazarl/go-bindata-assetfs" as assetfs
command/agent/http.go:199:102: undefined: assetFS
make: *** [pkg/darwin_amd64/nomad] Error 2
```

--- a/README.md
+++ b/README.md
@@ -137,10 +137,12 @@ https://github.com/google/protobuf for more information.
 
 **Note:** Building the Nomad UI from source requires Node, Yarn, and Ember CLI. These tools are already in the Vagrant VM. Read the [UI README](https://github.com/hashicorp/nomad/blob/master/ui/README.md) for more info.
 
-To cross-compile Nomad, run `make release`. This will compile Nomad for multiple
+To cross-compile Nomad, run `make prerelease` and `make release`.
+This will generate all the static assets, compile Nomad for multiple
 platforms and place the resulting binaries into the `./pkg` directory:
 
 ```sh
+$ make prerelease
 $ make release
 ...
 $ ls ./pkg

--- a/client/allocrunner/testing.go
+++ b/client/allocrunner/testing.go
@@ -1,3 +1,5 @@
+// +build !release
+
 package allocrunner
 
 import (

